cluster/gce/configure-vm.sh

```diff
@@ -140,15 +140,30 @@ function remove-docker-artifacts() {
   echo "== Finished deleting docker0 =="
 }
 
-# Retry a download until we get it.
+# Retry a download until we get it. Takes a hash and a set of URLs.
 #
-# $1 is the URL to download
+# $1 is the sha1 of the URL, if known. $2+ are the URLs to download.
```

Comment:
The comment should probably say "can be empty". It should take the empty argument $1.